src/invidious/database/channels.cr

```diff
@@ -28,6 +28,17 @@ module Invidious::Database::Channels
     PG_DB.exec(request, args: channel_array)
   end
 
+  def delete_not_subscribed
+    request = <<-SQL
+      DELETE FROM channels *
```

Comment:
This query takes AGES to finish on a big instance. After 300 seconds it hasn't finished completing. I do have 36232 users though.

Reply:
Maybe it's because it has never been done before? It took quite some time on the test instance too when I ran it...